From 9fa250d794cabc68fe53087de1b5b9797e515e18 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Sun, 1 Jan 2017 03:51:09 -0500
Subject: [PATCH 3/9] Move #include for type definition to correct place.

Not sure why the old way works in newer compilers; maybe they just
pre-define these types by defaulting to a newer standard.
---
 src/timestamp.h | 6 ++++++
 src/util.h      | 1 -
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/timestamp.h b/src/timestamp.h
index 58ae148..5565154 100644
--- a/src/timestamp.h
+++ b/src/timestamp.h
@@ -15,6 +15,12 @@
 #ifndef NINJA_TIMESTAMP_H_
 #define NINJA_TIMESTAMP_H_
 
+#ifdef _WIN32
+#include "win32port.h"
+#else
+#include <inttypes.h>
+#endif
+
 // When considering file modification times we only care to compare
 // them against one another -- we never convert them to an absolute
 // real time.  On POSIX we use timespec (seconds&nanoseconds since epoch)
diff --git a/src/util.h b/src/util.h
index a8493d6..4ee41a5 100644
--- a/src/util.h
+++ b/src/util.h
@@ -18,7 +18,6 @@
 #ifdef _WIN32
 #include "win32port.h"
 #else
-#include <inttypes.h>
 #include <stdint.h>
 #endif
 
-- 
2.14.1

